web-app/src/app/tabs/battery/battery.component.html

```diff
@@ -1,207 +1,231 @@
 <div class="battery">
 	<div class="header">
-		<span class="aliveGroup">
-	    		<div *ngIf="battery.batteryAlive; else elseBlock"><img class="circleAlive" src="../../../assets/GreenCircle.png" alt="Red Circle" id="circle"></div>
-	    		<ng-template #elseBlock><img class="circleAlive" src="../../../assets/RedCircle.png" alt="Red Circle" id="circle"></ng-template>
-				<span class="aliveText">Alive</span>
-		</span>
+		<div class="alive-group">
+	    		<div *ngIf="battery.batteryAlive; else elseBlock"><img class="circle" src="../../../assets/GreenCircle.png" alt="Red Circle" ></div>
+	    		<ng-template #elseBlock><img class="circle" src="../../../assets/RedCircle.png" alt="Red Circle" ></ng-template>
+				<span class="alive-text">Alive</span>
+		</div>
 	</div>
-	<div class="container-row full">
-		<div class="leftside">
-				<div class ="subheader">
-				<h3>BMS Relay Status Flags</h3>
+	<div class="container-row">
+		<div class="side">
+			<div class ="subheader">BMS Relay Status Flags</div>
+			<div class="container-row">
+				<div class="container-column">
+					<span class="item">
+						<span *ngIf="battery.bmsRelayStatusFlags.dischargeRelayEnabled; else elseBlock" class="ifs"><img class="circle-items" src="../../../assets/GreenCircle.png" alt="Red Circle" ></span>
```

Comment:
Is there any way to reduce the length of the conditionals? Also for the items it might be better to just make a list rather than a lot of spans.

Reply:
Found out how to reduce the conditionals, but I'm not sure how to make a list that cuts down on the code, do you have any ideas?

Reply:
If its needed that's okay I guess. I'm not familiar with grid layout but if you have time to check it out it could probably make things a lot more readable and easier. https://css-tricks.com/snippets/css/complete-guide-grid/